<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_login">
     <widget class="QLineEdit" name="lineEdit_cardNumber">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cardPin">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>490</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Card number</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pin code</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_loginStatus">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_main">
     <widget class="QPushButton" name="pushButtonShowBalance">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Show balance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogout">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>390</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Show history</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeposit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>480</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>480</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mainUsername">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_showbalance">
     <widget class="QPushButton" name="pushButtonBackShowBalance">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_showBalanceBalance">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Account balance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Account owner:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_showBalanceOwner">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_deposit">
     <widget class="QPushButton" name="pushButtonBackDeposit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDepositDeposit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>480</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_depositAmount">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>390</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_depositStatus">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>440</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deposit amount</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBankautomat">
    <property name="title">
     <string>Bankautomat</string>
    </property>
   </widget>
   <addaction name="menuBankautomat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
